org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/preferences/Preferences.java

```diff
@@ -830,6 +837,9 @@ public static Preferences createFrom(Map<String, Object> configuration) {
 		boolean signatureHelpEnabled = getBoolean(configuration, SIGNATURE_HELP_ENABLED_KEY, true);
 		prefs.setSignatureHelpEnabled(signatureHelpEnabled);
 
+		boolean signatureDescriptionEnabled = getBoolean(configuration, SIGNATURE_HELP_DESCRIPTION_ENABLED_KEY, true);
+		prefs.setSignatureHelpDescriptionEnabled(signatureDescriptionEnabled);
```

Comment:
I noticed we return `true` by default here if we can't locate the values in the configuration. Shouldn't it be `false` ? They're initialized as `false` when looking at the fields.